docs/guides/publisher-client-side.md

```diff
@@ -171,18 +187,20 @@ Based on the status and availability of a valid identity, the SDK sets up the ba
 You need to consider how you pass the returned advertising token to SSPs.
 :::
 
+(**GWH_LP04 is there any guidance at all that we can offer re the above? Maybe an example or something?**)
```

Comment:
The difficulty here is that our main advice is to use Prebid.js (in which case they shouldn't be following this guide) or Google Ad Manager Secure Signals (in which case they shouldn't be following this guide).
The only publishers using this guide should be those who *don't* want to follow our advice about how to pass the token to SSPs - probably because they already have a different system they use.

Perhaps some text to that effect might be helpful here?

Reply:
Got it. Thx. Here's what I put. Any input welcome.
You need to consider how you pass the returned advertising token to SSPs. With some other approaches to client-side UID2 implementation, such as using Prebid.js (see Prebid Integration Guide or Google Ad Manager Secure Signals (see Google Ad Manager Secure Signals Integration Guide), the implementation includes functions that manage passing the returned advertising token. If you're using the UID2 SDK for JavaScript you'll need to manage this yourself.